### PR TITLE
push kiali image to both docker and quay. operator just goes to quay

### PR DESCRIPTION
**Pull request by jmazzitelli**
@israel-hdez please look this over and see if it makes sense/works.

The one part I'm not sure about is:

```
withCredentials([usernamePassword(credentialsId: 'kiali-docker', passwordVariable: 'DOCKER_PASSWORD', usernameVariable: 'DOCKER_USER'), usernamePassword(credentialsId: 'kiali-quay', passwordVariable: 'QUAY_PASSWORD', usernameVariable: 'QUAY_USER')]) {
          sh "make -f ${backendMakefile} -C ${backendDir} backend-push-docker"
          dockerTag = sh(returnStdout: true, script: "sed -rn 's/^VERSION \\?= v(.*)/v\\1/p' ${backendDir}/Makefile").trim()
          quayTag = sh(returnStdout: true, script: "sed -rn 's/^VERSION \\?= v(.*)/v\\1/p' ${backendDir}/Makefile").trim()
```

Not sure what these dockerTag (and now quayTag) are used for.

### CODE REVIEW

**Comment by rhqci** [comment]
Jenkins CI: kiali-core-pr-e2e-test [#895](https://paste.fedoraproject.org/paste/GW-KgjLINrT10u1aaMwmNg)
  * :heavy_check_mark: run-kiali-e2e-tests [#[1694]](https://paste.fedoraproject.org/paste/eDRm-ZTbhGRQfesozxmVTg)

File: Makefile [hunk 1]
```diff
@@ -25,10 +25,12 @@ VERSION_LABEL ?= ${VERSION}
...
 GO_VERSION_KIALI = 1.8.3
 
-# Identifies the docker image that will be built and deployed.
-DOCKER_NAME ?= kiali/kiali
-DOCKER_VERSION ?= dev
-DOCKER_TAG = ${DOCKER_NAME}:${DOCKER_VERSION}
+# Identifies the container image that will be built and deployed.
+CONTAINER_NAME ?= kiali/kiali
+CONTAINER_VERSION ?= dev
+CONTAINER_TAG = ${CONTAINER_NAME}:${CONTAINER_VERSION}
```
Comment (FilipB) [inline Makefile]: Why having both CONTAINER_TAG and DOCKER_TAG? CONTAINER_TAG is not used anywhere.

Comment (jmazzitelli) [reply Makefile]: CONTAINER_TAG is used to define the base name for both DOCKER_TAG and QUAY_TAG - see the 2 lines after this

**Comment by rhqci** [comment]
Jenkins CI: kiali-core-pr-e2e-test [#903](https://paste.fedoraproject.org/paste/xuA1~-jyHzzZ5ZQh3YcSUQ)
  * :red_circle: run-kiali-e2e-tests [#[1700]](https://paste.fedoraproject.org/paste/y4SyKlQJKeFuNibQ~mIn0g)

**Comment by rhqci** [comment]
Jenkins CI: kiali-core-pr-e2e-test [#904](https://paste.fedoraproject.org/paste/3xNwCCO8uBOZlBXSd7n-Aw)
  * :red_circle: run-kiali-e2e-tests [#[1701]](https://paste.fedoraproject.org/paste/IRN9aKrYovf6cRWXf25mVA)

**Pull request merged**